docs/source/setup/introduction_to_alns.rst

```diff
@@ -0,0 +1,68 @@
+A brief introduction to ALNS
+============================
+
+Before we explain what ALNS is, we first need to describe the type of problems it can solve well.
+These problems typically come from the field of `combinatorial optimisation <https://en.wikipedia.org/wiki/Combinatorial_optimization>`_, and have
+
+* A large (but finite) set of solutions (the *search space*)
+* An objective to be optimised by finding a *good* solution in the search space
+
+A very well-known example of such a problem is the `travelling salesman problem <https://en.wikipedia.org/wiki/Travelling_salesman_problem>`_ (TSP), which asks
+
+    Given a list of :math:`n` cities and the distances between each pair of cities, what is the shortest possible route that visits each city exactly once and returns to the origin city?
+
+It is easy to find *a* solution: :math:`1 \rightarrow 2 \rightarrow 3 \rightarrow \ldots \rightarrow n \rightarrow 1` is one option.
+Finding an *optimal* solution, however, can be very difficult since there are :math:`n!` possible solutions, and the search space thus grows very quickly with :math:`n`.
+
+Because of the explosive growth of the search space, combinatorial problems cannot be solved optimally by testing every solution in the search space.
+In such cases, a heuristic method can help find good (but not necessarily optimal) solutions quickly.
+ALNS is one such heuristic method.
+
+.. note::
+
+    :ref:`/examples/travelling_salesman_problem.ipynb` solves a TSP instance using ALNS.
+
+Specifically, the adaptive large neighbourhood search metaheuristic is:
```

Comment:
```suggestion
Specifically, ALNS is:
```
- metaheuristic is defined later. Could also be replaced with “algorithm”.
- use of acronym (optional). The full name of ALNS is not mentioned before this point by the way. Maybe define this at the top?

Reply:
> The full name of ALNS is not mentioned before this point by the way.

No, but it is defined on the index page, and I think we can get away with not having to explain what ALNS stands for on every page of the `alns` documentation :-).

I'm explicit about the individual words here because the list below it explains each of them.